low-level/sensors/impl/generic/node_hw.py

```diff
@@ -972,6 +980,11 @@ def _parse_fan_info(self, index, date, _time, device_id, sensor_id,\
         fan_info = fan_specific_data
         fan_info.update({"fru_id" : device_id, "event" : event})
         resource_type = NodeDataMsgHandler.IPMI_RESOURCE_TYPE_FAN
+        is_fru, fru_type_unit = self.ipmi_client.is_fru(
+            self.fru_map[self.TYPE_FAN])
+        fru = False
+        if is_fru:
+            fru = str(is_fru) + "/" + fru_type_unit
```

Comment:
use : delimeter

Reply:
so, node_hw sensor & actuator both using same format.
should I keep the func in utility then?

Reply:
done.